base/sourcegraph/syntactic-code-intel-worker/worker.Deployment.yaml

```diff
@@ -0,0 +1,77 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  annotations:
+    description: High level syntax analysis
+  labels:
+    deploy: sourcegraph
+    sourcegraph-resource-requires: no-cluster-admin
+    app.kubernetes.io/component: syntactic-code-intel-worker
+  name: syntactic-code-intel-worker
+spec:
+  minReadySeconds: 10
+  replicas: 2
+  revisionHistoryLimit: 10
+  selector:
+    matchLabels:
+      app: syntactic-code-intel-worker
+  strategy:
+    rollingUpdate:
+      maxSurge: 1
+      maxUnavailable: 1
+    type: RollingUpdate
+  template:
+    metadata:
+      labels:
+        deploy: sourcegraph
+        app: syntactic-code-intel-worker
+    spec:
+      containers:
+        - name: syntactic-code-intel-worker
+          env:
+            - name: SYNTACTIC_CODE_INTEL_BACKEND
+              value: blobstore
+            - name: SYNTACTIC_CODE_INTEL_UPLOAD_AWS_ENDPOINT
+              value: http://blobstore:9000
+            - name: SYNTACTIC_CODE_INTEL_WORKER_ADDR
+              value: ":3188"
+            - name: POD_NAME
+              valueFrom:
+                fieldRef:
+                  fieldPath: metadata.name
+          image: index.docker.io/sourcegraph/syntactic-code-intel-worker:6.0.0@sha256:50bdeb38b196f0fc21404969016bf8263f78144292e905867e93480f66c8251c
```

Comment:
we have newer version than this?

Reply:
Yeah, this is the version that is in `main` right now though.